pytest_mpl/plugin.py

```diff
@@ -62,7 +62,7 @@ def _download_file(baseline, filename):
             u = urlopen(base_url + filename)
             content = u.read()
         except Exception as e:
-            warnings.warn('Downloading {0} failed: {1}'.format(base_url + filename, e))
```

Comment:
I would still like to at least log this, pytest must have a way of bubbling things up to the user?

Reply:
I guess you could have env var to set verbosity and then use `log` or `print` if it is set to verbose? I still want it off by default though.

Reply:
I get why we run pytest with all warnings as errors, but now we have taken away a thing which lets us tell the user something is going on. Saying that "this URL didn't work" is not something I am happy removing, we need to come up with a solution.

I can't see anything helpful on the pytest webiste about plugins logging info level stuff. :shrug:

Reply:
Looks like this might be as close as we can get: https://stackoverflow.com/questions/51737378/how-should-i-log-in-my-pytest-plugin

Reply:
See second commit.